app/css/application.css

```diff
@@ -18,8 +18,61 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
+:root {
+    /* TODO: Rename these to match figma */
+    --c-CBC9D9: #cbc9d9;
```

Comment:
These will be named to `darkBlue` etc. They cover the entire Exercism palette, both light and dark